.github/actions/deploy/tls_manager.py

```
@@ -0,0 +1,694 @@
+                    "readOnly": true
+                  }},
+                  {{
+                    "name": "ca-certs",
+                    "mountPath": "/etc/mysql/ca",
+                    "readOnly": true
+                  }}
+                ]
+              }}
+            ]'''
+
+            print(f'🔧 Patching MariaDB deployment to enable TLS...')
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)